src/PdfSmith/wwwroot/index.html

```diff
@@ -0,0 +1,133 @@
+                </div>
+            </div>
+        </section>
+
+        <section id="features" class="features">
+            <div class="container">
+                <h2>Key Features</h2>
+                <div class="features-grid">
+                    <div class="feature-card">
+                        <div class="feature-icon">⚡</div>
+                        <h3>Lightning Fast</h3>
+                        <p>Generate PDFs in seconds using high-performance Chromium engine with optimized rendering</p>
+                    </div>
+                    <div class="feature-card">
+                        <div class="feature-icon">🎨</div>
+                        <h3>Multiple Template Engines</h3>
+                        <p>Choose between Razor and Scriban template engines for maximum flexibility and power</p>
+                    </div>
+                    <div class="feature-card">
+                        <div class="feature-icon">🔐</div>
+                        <h3>Secure & Scalable</h3>
+                        <p>Built-in authentication, rate limiting, and request timeouts for production-ready deployment</p>
+                    </div>
+                    <div class="feature-card">
+                        <div class="feature-icon">📱</div>
+                        <h3>Responsive Design</h3>
+                        <p>Generate PDFs that look great on any device with responsive HTML templates</p>
+                    </div>
+                    <div class="feature-card">
+                        <div class="feature-icon">🌍</div>
+                        <h3>Internationalization</h3>
+                        <p>Full support for multiple cultures and languages in your PDF documents</p>
+                    </div>
+                    <div class="feature-card">
+                        <div class="feature-icon">🔧</div>
+                        <h3>Easy Integration</h3>
+                        <p>Simple REST API that works with any programming language and framework</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="api" class="api-example">
+            <div class="container">
+                <h2>Quick Start Example</h2>
+                <p style="text-align: center; margin-bottom: 2rem;">Send a POST request to generate your first PDF:</p>
+                
+                <div class="code-block">
+                    <strong>Endpoint:</strong> POST /api/pdf<br>
+                    <strong>Content-Type:</strong> application/json<br>
+                    <strong>Authorization:</strong> x-api-key: YOUR_API_KEY
+                </div>
+
+                <div class="code-block">
+{
+  "template": "&lt;html&gt;&lt;body&gt;&lt;h1&gt;Hello {{Model.Name}}!&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;",
+  "model": { "Name": "World" },
+  "templateEngine": "scriban",
+  "pdfOptions": {
+    "margin": { "top": "20", "bottom": "20", "left": "20", "right": "20" }
+  }
```

Comment:
The code block should isn't properly formatted: 
![image](https://github.com/user-attachments/assets/aba35c3d-b3e9-4074-806a-8eae8bbffe1a)

It should appear in this way:

```json
{
   "template":"<html><body><h1>Hello {{Model.Name}}!</h1></body></html>",
   "model":{
      "Name":"World"
   },
   "templateEngine":"scriban",
   "pdfOptions":{
      "margin":{
         "top":"20",
         "bottom":"20",
         "left":"20",
         "right":"20"
      }
   }
}
```